doc/classes/PhysicsDirectSpaceState2D.xml

```diff
@@ -109,12 +109,12 @@
 			<argument index="5" name="collide_with_areas" type="bool" default="false">
 			</argument>
 			<description>
-				Intersects a ray in a given space. The returned object is a dictionary with the following fields:
+				Intersects a ray in a given space. If the ray originates inside of one or more objects, the first object detected will be selected. The returned object is a dictionary with the following fields:
 				[code]collider[/code]: The colliding object.
 				[code]collider_id[/code]: The colliding object's ID.
 				[code]metadata[/code]: The intersecting shape's metadata. This metadata is different from [method Object.get_meta], and is set with [method PhysicsServer2D.shape_set_data].
-				[code]normal[/code]: The object's surface normal at the intersection point.
-				[code]position[/code]: The intersection point.
+				[code]normal[/code]: The object's surface normal at the intersection point. If the ray originates inside of the object, this will be a zero length [code]Vector2[/code].
```

Comment:
```suggestion
				[code]normal[/code]: The object's surface normal at the intersection point. If the ray originates inside the object, this will be a zero length [Vector2].
```

Reply:
Same thing for the other documented classes ("inside of" -> "inside", and use `[Class]` markup).